### PR TITLE
fix: copy cedana binary and install criu from deb package and yum

### PR DESCRIPTION
**Pull request by bsmithai**
## Describe your changes
- yum package: criu added
- copy cedana from container to `/host/usr/local/bin`
- build-start-daemon.sh with --nobuild

## Issue ticket number 
[CED-461](https://linear.app/cedana/issue/CED-461/versioning-in-cedana-controller)

## Checklist before requesting a review
- [x] Have I performed a self-review?
- [ ] Have I added regression or unit tests (where it makes sense) for my changes?
- [ ] Have I updated the README if changes have resulted in it being out of date?
- [ ]  Have I checked to ensure my PR only introduces the changes it's intended to? E.g. No extraneous formatting changes.
- [ ] Is this a breaking change? If yes, please describe areas affected and request reviews from developer stakeholders.

### CODE REVIEW

File: scripts/k8s/setup-host.sh [hunk 1]
```diff
@@ -28,11 +31,21 @@ install_yum_packages() {
     yum group install -y "Development Tools"
 }
 
+install_criu_ubuntu_2004() {
```
Comment (nravic) [inline scripts/k8s/setup-host.sh:34]: this says 2004, but what you download is for 22.04

**Review by nravic — APPROVED** [review]
small q, lgtm otherwise

File: scripts/k8s/setup-host.sh (hunk 1 above)
Comment (bsmithai) [reply scripts/k8s/setup-host.sh:34]: facts

File: cmd/scripts/k8s/setup-host.sh [hunk 2]
```diff
@@ -28,11 +31,21 @@ install_yum_packages() {
...
+    dpkg -i $OUTPUT_FILE
+    rm $OUTPUT_FILE
+}
+
 if [ -f /etc/os-release ]; then
     . /etc/os-release
     case "$ID" in
         debian | ubuntu)
             install_apt_packages
+            install_criu_ubuntu_2004
```
Comment (nravic) [inline cmd/scripts/k8s/setup-host.sh:48]: old function name

File: cmd/scripts/k8s/setup-host.sh [hunk 3]
```diff
@@ -1,12 +1,15 @@
 #!/bin/bash
 
+# Install Cedana
+cp /usr/local/bin/cedana /host/usr/local/bin/cedana
+
```
Comment (nravic) [inline cmd/scripts/k8s/setup-host.sh:5]: also copy the build-start-daemon from where it was extracted by the dockerfile

**Review by nravic — COMMENTED** [review]
uno mas

File: cmd/scripts/k8s/setup-host.sh (hunk 3 above)
Comment (bsmithai) [reply cmd/scripts/k8s/setup-host.sh:5]: just copy it to /host/build-start-daemon.sh ?

Comment (nravic) [reply cmd/scripts/k8s/setup-host.sh:5]: yeah looks like that's where the setup script is running build-start-daemon from

Comment (nravic) [reply cmd/scripts/k8s/setup-host.sh:5]: oh wait we still do a git clone nvm then

**Review by nravic — APPROVED** [review]
lgtm

**Pull request merged**